src/GitUtilities.js

```diff
@@ -56,6 +62,16 @@ export default class GitUtilities {
   }
 
   @logger.logifySync()
+  static getLastTag() {
+    try {
+      return ChildProcessUtilities.execSync("git describe --tags --abbrev=0");
+    } catch (e) {
+      return "0.0.0";
```

Comment:
Removing out this try catch, added check case in updated packages collector. No need to handle it here then. It didn't even make sense to try catch it.